rlkit/torch/model_based/dreamer/visualization.py

```diff
@@ -81,10 +92,12 @@ def visualize_rollout(
                 else:
                     policy_o = observation.reshape(1, -1)
                 # hack to pass typing checks
+                # TODO why reshape here?
```

Comment:
please remove unnecessary comments

Reply:
Why is the reshape required though? What are you trying to exactly do here?

Reply:
the policy expects batched obs thats all

Reply:
world model as well

Reply:
Ah that makes sense.